extensions/scala/probePlugin/src/main/scala/org/virtuslab/intellij/scala/probe/SbtProbeHandlerContributor.scala

```diff
@@ -0,0 +1,14 @@
+package org.virtuslab.intellij.scala.probe
```

Comment:
let's use org.virtuslab.ideprobe.scala package for this extension.

Reply:
done